src/emoji.cr

```diff
@@ -15,7 +16,12 @@ module Emoji
     s
   end
 
-  def self.sanitize(s : String)
-    s.gsub(EMOJI_REGEX, "")
+  def self.sanitize(s : String, regex = :simple)
```

Comment:
Is it worth using enum here?

Reply:
Kind of off-topic but I'd replace `s` with `str` to be at least bit more descriptive ;)

Reply:
I would prefer to keep it simple here. Regular symbol works just fine IMO

Reply:
With autocasting you still should be able to use them, having compile-time safety without manual checks :) but that's optional for me, LGTM in any case.